src/platform/imx8/include/platform/drivers/interrupt.h

```diff
@@ -43,6 +43,8 @@
 /* 32 HW interrupts + 8 IRQ_STEER lines each with 64 interrupts */
 #define PLATFORM_IRQ_HW_NUM	XCHAL_NUM_INTERRUPTS
 #define PLATFORM_IRQ_CHILDREN	64
+/* The cascaded struct must be modulo 64.*/
+#define PLATFORM_IRQ_FIRST_CHILD 0
```

Comment:
`+#define PLATFORM_IRQ_FIRST_CHILD 0` might not work

Reply:
having a look

Reply:
I have updated the commit. The range [0-63] being of no use for DSP on IMX8 and would require a small fix in the SOF interrupt.c to allow the overlap with the HW interrupts, it's better for the moment to start at 64, as we did together. 
I just have tested that new version of the fix on our SAI test branch and it works fine.